app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require("supertest");
+const seed = require("./db/seeds/seed.js");
+const data = require("./db/data/test-data");
+const app = require("./app.js");
+const db = require("./db/connection.js");
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('getCategories', () => {
+    test("Responds with an object containing an array of category objects", () => {
+        return request(app)
+          .get("/api/categories")
+          .expect(200)
+          .then((res) => {
+            expect(res.body.categories).toEqual(expect.any(Array));
```

Comment:
This test asserts that the data type is an array, but the next test uses a forEach that is also making sure that the data is an array. That means that we don't need this first test.
This means it's an extra request that isn't really necessary, not a deal-breaker but worth keeping in mind for future tests.